Add boefje that checks witha.name for DDOS targets. (#4443)
minvws/nl-kat-coordination: boefjes/boefjes/plugins/kat_withaname/description.md

> # Ddosia / A bot a name

Reviewer: ```suggestion
# DDoSia / A bot a name
```

> This uses the A bot With a name list to detect if our ip's hostnames are listed for Ddosia to attack.

Reviewer: ```suggestion
This uses the [WhitAName!](https://witha.name) list to detect if our IPs hostnames are listed for DDoSia to attack.
```
